README.md

```diff
@@ -40,6 +40,20 @@ $ cmake --build .
 This will build Fizzy as a library and since there is no public API
 (the so called *embedder API* in WebAssembly) yet, this is not very useful.
 
+## WASI
+
+Building with the `FIZZY_WASI` option will output a `fizzy-wasi` binary implementing
+the [WASI] API (a very limited subset of `wasi_snapshot_preview1`, to be precise).
+It uses [uvwasi] under the hood. It can be used to execute WASI-compatible binaries on the command line.
+
+```sh
+$ mkdir build && cd build
+$ cmake -DFIZZY_WASI=ON ..
+$ cmake --build .
+```
+
```

Comment:
Perhaps we could add an example of `fizzy-wasi` usage like
```
$ bin/fizzy-wasi ../tools/wasi/smoketest/helloworld.wasm 
Hello, world!
```

Reply:
Done.